### ISSUE TITLE
wrong paste chars

### ISSUE THREAD

**Issue by witchanto** [joncampbell123/dosbox-x#4336]
### Describe the bug

if i press ctrl+f6 than will be inserted chars that are not in the win 10 -clipboard

### Steps to reproduce the behaviour

windows 10: ctrl+ins
dos-box-x: ctrl+f5

### Expected behavior

to written the text from windows-clipboard

### What operating system(s) this bug have occurred on?

win 10 22h2

### What version(s) of DOSBox-X have this bug?

0.83.25 sdl2

### Have you checked that no similar bug report(s) exist?

- [X] I have searched and didn't find any similar bug report.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by witchanto** [comment]
i do not think that it has to do with language because that are chars which are unusual. no part is a part of the clipboard.

**Comment by rderooy** [comment]
I just tried pasting on Linux SDL2 with CTRL-F6 and it works just fine. Changing keyboard layout has no apparent effect, it still pastes the text correctly.

**Comment by maron2000** [comment]
@rderooy Thank you for testing on linux. It seems that something is wrong or some limitation on Windows build.

**Comment by rderooy** [comment]
Perhaps @witchanto can try to paste using the menu item (Main -> Shared Clipboard Functions -> Paste from the clipboard), or the CTRL-F6 shortcut. It may be an issue with the CTRL-Ins keyboard shortcut, which does not do anything on Linux.

**Comment by Torinde** [comment]
I tried on Windows (but with latest build 2023.05.01, not with 0.83.25) - no problem for Latin letters. For other letters as maron2000 said - DOS codepage has to match the language of the letters you paste from Windows.

@witchanto, please share here the exact string you copy in Windows and how it appears in DOSbox. Both screenshots and the text itself.
